homeassistant/components/boschshc/strings.json

```diff
@@ -0,0 +1,25 @@
+{
+  "title": "Bosch Smart Home Controller",
+  "config": {
+    "step": {
+      "user": {
+        "title": "SHC access parameters",
+        "data": {
+          "name": "Name",
+          "ip_address": "IP Address",
```

Comment:
Please use as many common strings as possible.

Here are the common strings:
https://github.com/home-assistant/core/blob/4170eb0f83ee5fef11eef1b656d3aa4696c67b95/homeassistant/strings.json#L20-L55

Here is an example:
https://github.com/home-assistant/core/blob/4170eb0f83ee5fef11eef1b656d3aa4696c67b95/homeassistant/components/adguard/strings.json#L7